gammapy/spectrum/utils.py

```diff
@@ -13,19 +16,254 @@
            'np_to_pha',
            ]
 
-# TODO: remove functions now that we have classes!
 
+def find_log_centers(bin_edges):
+    """Compute energy bin log centers.
+
+    Bin center = sqrt(low_edge * high_edge)
+    """
+    return np.sqrt(bin_edges[:-1] * bin_edges[1:])
+
+
+def find_log_edges(bin_centers):
+    """Compute equally log-spaced energy bin edges
+    """
+
+    bin_edges = np.sqrt(bin_centers[:-1] * bin_centers[1:])
+    first = bin_centers[0] * bin_centers[0] / bin_edges[0]
+    last = bin_centers[-1] * bin_centers[-1] / bin_edges[-1]
+    bin_edges = np.append(first, bin_edges)
+    bin_edges = np.append(bin_edges, last)
+
+    return bin_edges
+
+
+class EnergyBinning(object):
+
+    """Class to handle energy dimension for e.g. counts spectra, ARF tables
+    and so on. The idea is to handel everything related to energy in this
+    class. It should probably be move somewhere else (not spectrum.utils)
+    rather datasets.energyhandler or similar. All other classes should then
+    takes this class as argument.
+
+    TODO:
+    - document once we agreed on a mechanism
+    - implement FITS I/O
+
+ 
+    Parameters
+    ----------
+    bin_edges : `~astropy.units.Quantity`
+    Energy bin edges
```

Comment:
This text description for a paramter should be indented by four more spaces (everywhere)